src/test/java/com/epam/tc/hw7/utils/JsonReaderForDatas.java

```diff
@@ -0,0 +1,50 @@
+package com.epam.tc.hw7.utils;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+public class JsonReaderForDatas {
+
+    public static Object[][] getDatas() {
+        try {
+            JSONParser parser = new JSONParser();
+            JSONObject datasFromJson = (JSONObject) parser.parse(new FileReader(
+                "src/test/resources/JDI_ex8_metalsColorsDataSet.json"));
+            return getObjectsData(datasFromJson);
+        } catch (IOException | ParseException e) {
+            e.printStackTrace();
+        }
+        return null;
+    }
+
+
+    private static Object [][] getObjectsData(JSONObject datasFromJson) {
+        Object [][] datas = {};
+        int nextDataNumber = 1;
+
+        while (datasFromJson.get("data_" + nextDataNumber) != null) {
```

Comment:
Very slow and difficult logic for reading: you just need only read all data objects get content

Reply:
No comments and no changes

Reply:
But we need to transfer the read JSON to DataProvider that needed Object [][] format. So, this method for that transition.

Reply:
And you can just iterate throw all data and get inner data object

Reply:
I iterate through all data and  get inner data object in my methods. I do not fully understand what you want me to write. Can you give me a hint - which way to look better (what method for JSONObject use)? Because the only way I found was to use JSONObject.get("test case name)  to get one test case object and put it to Object [][]